src/WormholeReceiver.sol

```diff
@@ -51,4 +57,10 @@ abstract contract WormholeReceiver is Ownable, WormholeBase {
     }
     _;
   }
+
+  modifier replayProtect(bytes32 deliveryHash) {
+    if (seenDeliveryVaaHashes[deliveryHash]) revert AlreadyProcessed(deliveryHash);
```

Comment:
random thought, since this `seenDeliveryVaaHashes` map is on each WormholeReceiver, is there any valid attack where someone sends a message w/ same `deliveryHash` to more than one receiver?